### ISSUE TITLE
Poly Gate input for Chord-Key

### ISSUE THREAD

**Issue by vcvrackideas** [MarcBoule/ImpromptuModular#6]
Hey Marc! Thanks for the new awesome Chord-Key module! Now it's pretty handy to make chord even for people that not pretty much into music theory) 
And also could it be possible to add ability to connect poly cable with gates to the Gate input? For example it can be used for making advanced sequences such as interval playing (for example 1-2 together and than 3-4 also together) and other creative things. Hope that it sounds not so complex)

**Comment by MarcBoule** [comment]
Interesting idea, thanks :-) I've just added it if ever you want to try it out. The algorithm is like this: for an output N, it takes the gate input poly channel N, or the highest one if the poly cable does not go up to N. 

So when giving a mono gate, all gates will be triggered. When giving a poly gate with 4 channels, each channel will trigger separately. If we give a poly gate with 2 channels, the top output will have a gate according to channel 1's input gate, but all other outputs will use the input gate on channel 2. Hope that makes sense!

By the way, did you solve your hotkey issue with Radeon Relive?

**Comment by vcvrackideas** [comment]
Thanks Marc! You just have unlocked my creativity decay with this module and this enhancement! 
I decided to not get you bothered by asking anything about Relive just because I use sync by waveform function in premiere\resolve. ) The last thing I got was unable to pick exactly same hotkey as relive already had. So it's hard to make you know what's happening actually. Personally I think that relive has some special "rights" on binded hotkey because it's always on after windows loaded just because it works from radeon's driver but I really dunno.
